script/prerender/createRedirects.ts

```diff
@@ -0,0 +1,43 @@
+import fs from 'fs';
+import path from 'path';
+import { Redirects } from '../../data/redirects/types';
+import { content } from '../../src/app/content/routes';
+import { makeUnifiedBookLoader } from '../../src/app/content/utils';
+import { findArchiveTreeNodeById } from '../../src/app/content/utils/archiveTreeUtils';
+import { AppServices } from '../../src/app/types';
+import { writeAssetFile } from './fileUtils';
+
+const redirectsPath = path.resolve(__dirname, '../../data/redirects/');
+
+const createRedirects = async(archiveLoader: AppServices['archiveLoader'], osWebLoader: AppServices['osWebLoader']) => {
+  const bookLoader = makeUnifiedBookLoader(archiveLoader, osWebLoader);
+
+  const books = fs.readdirSync(redirectsPath).filter((name) => name.match('.json'));
+
+  const redirects: Array<{ from: string, to: string }> = [];
+
+  for (const bookName of books) {
+    const bookRedirects: Redirects = require(redirectsPath + '/' + bookName);
+    const bookId = bookName.replace('.json', '');
+    const { tree, slug: bookSlug } = await bookLoader(bookId);
```

Comment:
you want to use the current version here, omitting it will get the _latest_ version which we may not have adopted yet, and the ability to not pass a version will go away when we adopt the new web pipeline archive